src/pages/recipes/index.tsx

```diff
@@ -71,17 +62,17 @@ export default function Recipes() {
             ? `Recipe statistics cached in Aiven for Redis®, retrieved in ${recipeStatsData.endToEndRetrievalTimeMs}ms (end-to-end with round
           trip between your serverless function and your database, results will vary depending on the region of your Aiven
           for Redis® instance).`
-            : "Recipe statistics retrieved from PostgreSQL database. To get cached results using Aiven for Redis®, please follow the instructions to set up your Redis instance."
+            : 'Recipe statistics retrieved from PostgreSQL database. To get cached results using Aiven for Redis®, please follow the instructions to set up your Redis instance.'
         }
       />
       {recipeStatsLoading ? (
         <DataTable.Skeleton columns={4} rows={7} />
       ) : recipeStatsError ? (
         <EmptyState
-          title={recipeStatsError.message ?? "Failed to load recipe stats."}
+          title={recipeStatsError.message ?? 'Failed to load recipe stats.'}
           primaryAction={{
             onClick: refetchRecipeStats,
-            text: "Refetch recipe stats",
+            text: 'Refetch recipe stats',
```

Comment:
this component has two nested ternary operations (one for stats + one for all recipes) that are quite similar to each other. I think we can make them into a component that handles:
1. loading state
2. empty state
3. table component

The table components can be separated into the `/components` folder, maybe `TableRecipes` and `TableRecipesStats`? We can also replace the nested ternaries with early conditional returns -- this is a personal preference because I think it's more readable :D

Reply:
fixed